lib/streamlit/connections/snowpark_connection.py

```diff
@@ -162,11 +167,24 @@ def _query(sql: str) -> pd.DataFrame:
 
         return _query(sql)
 
+    def session(self) -> "Session":
```

Comment:
should it be a `@property`?

Reply:
Whoops -- I missed this in the poll, will fix